green_cli/liquid.py

```diff
@@ -81,5 +107,59 @@ def sendtoaddress(session, details):
 green_cli.common.getbalance.params.append(confidential_option)
 green_cli.common.getunspentoutputs.params.append(confidential_option)
 
+def _print_tx_summary(tx):
+    click.echo(f"user signed: {tx['user_signed']}")
+    click.echo(f"server signed: {tx['server_signed']}")
+    click.echo(f"send all: {tx.get('send_all', False)}")
+    click.echo(f"utxo strategy: {tx['utxo_strategy']}")
+
+    available_per_asset = defaultdict(int)
+    used_per_asset = defaultdict(int)
+    change_per_asset = defaultdict(int)
+    for asset in tx['utxos']:
+        available = sum([utxo['satoshi'] for utxo in tx['utxos'][asset]])
+        available_per_asset[_asset_name(asset)] += available
+    for utxo in tx['used_utxos']:
+        used_per_asset[_asset_name(utxo['asset_id'])] += utxo['satoshi']
+    for asset in tx['change_amount']:
+        change_per_asset[_asset_name(asset)] += tx['change_amount'][asset]
+
+    for asset in available_per_asset:
+        click.echo(f"{asset}:")
+        click.echo(f"\tavailable: {available_per_asset[asset]}")
+        click.echo(f"\tused: {used_per_asset[asset]}")
+        click.echo(f"\tchange: {change_per_asset[asset]}")
+
+    click.echo(f"size: {tx['transaction_size']}")
+    click.echo(f"vsize: {tx['transaction_vsize']}")
+    click.echo(f"weight: {tx['transaction_weight']}")
+    click.echo(f"fee: {tx['fee']}")
+    click.echo(f"fee rate: {tx['calculated_fee_rate']} sat/kb")
+
+green_cli.tx._print_tx_summary = _print_tx_summary
+
+def _print_tx_output(options, output):
+    fg = None
+    if output['is_fee']:
+        fg = 'red'
+        if not options['show_all'] and not options['show_fee']:
+            return
+    elif output['is_change']:
+        fg = 'green'
+        if not options['show_all'] and not options['show_change']:
+            return
+    else:
+        if not options['show_all'] and options['show_fee'] or options['show_change']:
+            return
+
+    value = output['satoshi']
+    asset = _asset_name(output['asset_id'])
```

Comment:
nit `asset_name` is a better var name here IMO

Reply:
done, thanks